setup.py

```diff
@@ -80,16 +80,27 @@
 if has_cython:
     cmdclass['build_ext'] = build_ext
 
+runtime_lib_dirs =  os.environ['TA_LIBRARY_PATH']
+if runtime_lib_dirs:
+    runtime_lib_dirs = runtime_lib_dirs.split(os.pathsep)
+else:
+    runtime_lib_dirs = []
+
+# warnings.warn('runtime_lib_dirs = {}'.format(runtime_lib_dirs))
```

Comment:
Prefer to not have commented out stuff like this in the final patch that I would merge..